src/routes/(app)/collections/cookbooks/+page.svelte

```diff
@@ -85,9 +85,16 @@
 							class="items-center justify-center flex flex-row rounded-lg border-2 border-emerald-500 dark:bg-emerald-700 bg-emerald-100 p-1 dark:hover:bg-emerald-800 hover:bg-emerald-300"
 							><Save /></button>
 						<button
-							on:click={() => {
+							on:click={async () => {
 								currentCookbook = uneditedCookbook;
 								editCookbook = false;
+								const index = cookbooks.indexOf(currentCookbook);
+								cookbooks.splice(index, 1);
+
+								await fetch('/api/cookbook', {
```

Comment:
This is not going to work. What happens if a user just wants to edit a cookbook, and then they cancel the edit? It will delete the whole cookbook even though that is not what they wanted to do